packages/sparqlee/test/util/TestTable.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint mocha/no-exports: 0 */
```

Comment:
eslint sees this file as a test file and does not want it to export. I don't know thy it does this? Since the test pattern regex ` testRegex: ['/test/.*-test.*.ts$', '/test/.*\.test\.ts$', '/test/.*\.spec\.ts$']` should not match? (I even tred renaming the file to `EvaluationTable.ts` but this did not fix it either.